packages/orchestration/src/contracts/localchainAccountHolder.js

```diff
@@ -0,0 +1,154 @@
+// @ts-check
+/** @file Use-object for the owner of a localchain account */
+import { AmountShape } from '@agoric/ertp';
+import { makeTracer } from '@agoric/internal';
+import { UnguardedHelperI } from '@agoric/internal/src/typeGuards.js';
+import { M, prepareExoClassKit } from '@agoric/vat-data';
+import { TopicsRecordShape } from '@agoric/zoe/src/contractSupport/index.js';
+import { E } from '@endo/far';
+
+const trace = makeTracer('LCAH');
+
+const { Fail } = assert;
+/**
+ * @typedef {object} LocalChainAccountNotification
+ * @property {string} address
+ */
+
+/**
+ * @typedef {{
+ *   topicKit: import('@agoric/zoe/src/contractSupport/recorder.js').RecorderKit<LocalChainAccountNotification>;
+ *   account: import('@agoric/vats/src/localchain.js').LocalChainAccount | null;
+ * }} State
+ */
+
+const HolderI = M.interface('holder', {
+  getPublicTopics: M.call().returns(TopicsRecordShape),
+  makeDelegateInvitation: M.call(M.string(), AmountShape).returns(M.promise()),
+  makeCloseAccountInvitation: M.call().returns(M.promise()),
+  makeTransferAccountInvitation: M.call().returns(M.promise()),
+});
+
+/** @type {{ [name: string]: [description: string, valueShape: Pattern] }} */
+const PUBLIC_TOPICS = {
+  account: ['Account holder status', M.any()],
+};
+
+/**
+ * @param {import('@agoric/ertp').Baggage} baggage
+ * @param {import('@agoric/zoe/src/contractSupport/recorder.js').MakeRecorderKit} makeRecorderKit
+ * @param {ZCF} zcf
+ */
+export const prepareAccountHolder = (baggage, makeRecorderKit, zcf) => {
+  const makeAccountHolderKit = prepareExoClassKit(
+    baggage,
+    'Account Holder',
+    {
+      helper: UnguardedHelperI,
+      holder: HolderI,
+      invitationMakers: M.interface('invitationMakers', {
+        Delegate: HolderI.payload.methodGuards.makeDelegateInvitation,
+        CloseAccount: HolderI.payload.methodGuards.makeCloseAccountInvitation,
+        TransferAccount:
+          HolderI.payload.methodGuards.makeTransferAccountInvitation,
+      }),
+    },
+    /**
+     * @param {import('@agoric/vats/src/localchain.js').LocalChainAccount} account
+     * @param {StorageNode} storageNode
+     * @returns {State}
+     */
+    (account, storageNode) => {
+      // must be the fully synchronous maker because the kit is held in durable state
+      const topicKit = makeRecorderKit(storageNode, PUBLIC_TOPICS.account[1]);
```

Comment:
Is anything being published to vstorage yet? A requirement is not specified in the original ticket, but it seems like something we should tackle (perhaps, a follow up ticket). Could be a good addition to [this test](https://github.com/Agoric/agoric-sdk/pull/9078/files#diff-af9be96f19d77ebd6d0739b11b353aea50fd9dda04677403b8081ec50333058bR69-R71) - we could check delegatorAddress against something published to vstorage.

This likely needs design and additional input, but as a rough sketch we might want to do something like - 
```json
{
    "address": "agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce",
    "pub_key": {
      "@type": "/cosmos.crypto.secp256k1.PubKey",
      "key": "AgX1pQy65LmFCazXlahDxmgoDXxaiVFRwOnxj5Wo8ZCo"
    },
    "account_number": "138",
    "sequence": "404"
  }
  ```